lms/templates/courseware/progress.html

```diff
@@ -117,7 +117,12 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
                 <div class="requirement-container" data-eligible="${credit_course_requirements['eligibility_status']}">
                 %for requirement in credit_course_requirements['requirements']:
                     <div class="requirement">
-                        <div class="requirement-name">${_(requirement['display_name'])}</div>
+                        <div class="requirement-name">
+                            ${_(requirement['display_name'])}
+                            %if requirement['namespace'] == 'grade':
+                                <span>&nbsp;${int(requirement['criteria']['min_grade'] * 100)}%</span>
```

Comment:
I don't think you have to manually add space character `&nbsp;` here. It will be added by default.